CDS/scripts/atom_image.pl

```diff
@@ -30,21 +31,36 @@ sub is_imageurl_valid {
 print "INFO: Attempting to get image URL from CAPI\n";
 
 my $ua = LWP::UserAgent->new;
-my $response = $ua->request(GET 'https://internal.content.guardianapis.com/atom/media/'.$store->substitute_string($ENV{'atom_id'}));
 
-if ($response->code == 503) {
-	my $response = $ua->request(GET 'https://internal.content.guardianapis.com/atom/media/'.$store->substitute_string($ENV{'atom_id'}));
-}
-
-if ($response->code != 200) {
-	print "-ERROR: Error when attempting to access CAPI \n";
-	exit(1);
+my $retries = 0;
+
+my $response;
+while(true) {
+	$response = $ua->request(GET 'https://internal.content.guardianapis.com/atom/media/'.$store->substitute_string($ENV{'atom_id'}));
+	if($response->code >=400 && $response->code <=499){
+     	print $response->content;
+     	print "\n-ERROR: CAPI returned " . $response->code . ", actual error is logged above.\n";
+     	exit(1);
+  	} elsif ($response->code >=500 && $response->code <= 599){
+     	print $response->content;
+     	print "\n-ERROR: CAPI returned " . $response->code . ", actual error is logged above.\n";
+     	exit(1) if($retries>=8);
```

Comment:
please could you make `$max_retries` and `$sleep_delay` parameters configurable from the XML file, with the values here as "sensible defaults".

also I'd appreciate it if you could sort the indentation 😄

Reply:
Done